test/test-mc-tokens.c

```diff
@@ -152,6 +157,20 @@ static void _mc_token_test_run(_mongocrypt_tester_t *tester, const char *path) {
     TEST_DERIVED(ECC)
 #undef TEST_DERIVED_FROM_DATA_TOKEN
 
+// (EDC/ESC/ECC)TwiceDerivedToken(Tag|Value)?
+#define TEST_TWICE(Name, Suffix)                                                                                       \
+    mc_##Name##TwiceDerived##Suffix##_t *Name##TwiceDerived##Suffix =                                                  \
+        mc_##Name##TwiceDerived##Suffix##_new(crypt->crypto, Name##DerivedFromDataTokenAndCounter, status);            \
+    ASSERT_OR_PRINT(Name##TwiceDerived##Suffix, status);                                                               \
+    ASSERT_CMPBUF(*mc_##Name##TwiceDerived##Suffix##_get(Name##TwiceDerived##Suffix),                                  \
+                  test.Name##TwiceDerived##Suffix);
+   TEST_TWICE(EDC, Token);
+   TEST_TWICE(ESC, TagToken);
+   TEST_TWICE(ESC, ValueToken);
+   TEST_TWICE(ECC, TagToken);
+   TEST_TWICE(ECC, ValueToken);
```

Comment:
Run `./etc/format-all.sh` to format and fix the `check-format` task.

Reply:
Ah, that's the script I was looking around for. Thanks!